database/alembic.ini

```diff
@@ -61,7 +61,7 @@ version_path_separator = os  # Use os.pathsep. Default configuration used for ne
 # are written from script.py.mako
 # output_encoding = utf-8
 
-sqlalchemy.url = postgresql://mac:D100406m@localhost:5432/SoftProg
+sqlalchemy.url = postgresql://postgres:Vladik@localhost:5432/SportHUYnya
```

Comment:
Please revert this. These changes are not yours - so they have no business being in your PR even though they seem harmless. You waste reviewer's time with those random changes all over your PRs.